photon-client/src/views/CamerasView.vue

```diff
@@ -505,6 +502,17 @@ export default {
                 return this.$store.getters.currentPipelineIndex === -2;
             }
         },
+        resolutionList: {
+            get() {
+                let temp_dict = {};
+                let videoFormatList = this.$store.getters.videoFormatList;
+                for(let i = 0; i < videoFormatList.length; i++) {
+                  let videoFormat = videoFormatList[i];
+                  temp_dict[videoFormat["width"]+" X "+videoFormat["height"]] = i;
+                }
+                return temp_dict;
+            }
+        },
```

Comment:
Where's this function called from?

Reply:
Good observation :facepalm:. Found the actual issue: the selectedFilterResIndex set function mutates the calibration state to the correct index by mapping it to filteredResolutionList. When the page first loads, the UI automatically puts the default value, but it doesn't run the selectedFilterResIndex set method and thus doesn't update the calibration state correctly. Creating a solution.